### ISSUE TITLE
Quote is visible in class name when using hide circle

### ISSUE THREAD

**Issue by krisztianb** [plantuml/plantuml#342]
Hi there!

It seems that I found a small bug. Here is a simple PlantUML example to reproduce it.

This is fine:

````
@startuml
class "EventEmitter"
@enduml
````

As it creates this output:

![OK](http://www.plantuml.com/plantuml/svg/SoWkIImgAStDuKhEIImkLb1oBKlDAt7DpImfIIrIud98pKi1IW80)

But when I use `hide circle` to hide the circled char before the class name:

````
@startuml
hide "EventEmitter" circle
class "EventEmitter"
@enduml
````

It looks like this:

![Wrong](http://www.plantuml.com/plantuml/svg/SoWkIImgAStDuSh8J4bLK78jIyqhSStDB2b9BL9IICuiIiv9vKhEIImkHfFZSaZDIm7g0m00)

Can you spot the error? The quotes are visible in the class name.

Maybe I can help you fix this if you can point me into the right direction?

**Comment by krisztianb** [comment]
**Update:** The quotes are not visible if `hide "EventEmitter" circle` is used after the class definition. Strange...

**Comment by krisztianb** [comment]
It looks like this has been fixed in [V1.2020.15 (28 Jun, 2020)](https://plantuml.com/changes). So closing this issue.

Thanks for the very fast fix. 👍
